### PR TITLE
Prayer times

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary
Intitial preview of prayer times feature

image

image

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/index.module**](https://quran-com-bz9dsg80a-qurancom.vercel.app/index.module) | [**/**](https://quran-com-bz9dsg80a-qurancom.vercel.app/) |
|:-:|:-:|
|<a href="https://quran-com-bz9dsg80a-qurancom.vercel.app/index.module">Screenshot of /index.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/C9qZeW_yDFRz0OMwclo9hswW7X--oEFAfdPYeqzskEt9doJb9jiXad7tVw6CwW3mwcowCGocubGuxIqhRKuohg.png">(view full size)</a> |<a href="https://quran-com-bz9dsg80a-qurancom.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/REc0Hy0dgs1xo5AZ8fBvNXsKxQ3-1djiHJHsNcxRmRCbwbIu-51lK2bN9kr7Dy4wI3r1MGkq5Qg_DE_Kr_QJKw.png">(view full size)</a> |







Commit 6d2939d9ac11674f08482174a8e8cbc58469b464 (https://quran-com-bz9dsg80a-qurancom.vercel.app).

File: src/components/HomePage/HomePageImage.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import homepageImage from 'public/images/homepage.png';
+
+const HomePageImage = () => {
+  return (
+    <div className={styles.homepageImageContainer}>
+      <Image
+        className={styles.homepageImage}
+        src={homepageImage}
+        objectFit="cover"
+        sizes="(max-width: 425px) 95vw, 80vw"
```
Comment (tifa2UP) [inline src/components/HomePage/HomePageImage.tsx:15]: while you're at it, can you fix image pixelation on mobile

File: src/components/HomePage/PrayerTimes.module.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  width: 100%;
+  right: 0;
+  left: 0;
+  box-sizing: border-box;
+  padding: var(--spacing-medium);
+  position: absolute;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  background: linear-gradient(180deg, #000000a6, transparent);
```
Comment (tifa2UP) [inline src/components/HomePage/PrayerTimes.module.scss:11]: Make it a const

File: src/components/HomePage/PrayerTimes.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    month: 'long',
+    weekday: 'long',
+    year: 'numeric',
+  })
+    .format(Date.now())
+    .split(',')
+    .slice(1)
+    .join(',')
+    .trim();
+};
```
Comment (tifa2UP) [inline src/components/HomePage/PrayerTimes.tsx:68]: Need to double check the hijri date

**Review by tifa2UP — APPROVED** [review]

File: src/components/HomePage/PrayerTimes.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+const getNextPrayerTimes = (
```
Comment (tifa2UP) [inline src/components/HomePage/PrayerTimes.tsx:70]: this should properly format the time, should 12:01 instead of 12:1

**Pull request merged**